File: 8-kyu/Fake Binary.py
# DESCRIPTION:
# Given a string of digits, you should replace any digit below 5 with '0' and any digit 5 and above with '1'. Return the resulting string.
#
# Note: input will never be an empty string
def fake_bin(x):
    # zero = str(x)
    num = list(x)
    new = []
    res = ''
    for i in num:
        # int(i)
        if int(i)>=5: new.append('1')
        else: new.append('0')
    for x in new:
        res +=x

    return res
print(fake_bin('2312'))

File: JavaScript/8-kyu/Fake Binary.js
// Given a string of digits, you should replace any digit below 5 with '0' and any digit 5 and above with '1'. Return the resulting string.

// Note: input will never be an empty string

function fakeBin(x){
    let a = (x.toString()).split('');
    let result = '';
        a.forEach(element => {
            if(parseInt(element)<5){
                result+=0;
            } else result+=1;
        });
        return result;
}
console.log(fakeBin(14324234666));
